wallet/src/ui/app/components/sui-apps/index.tsx

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import cl from 'classnames';
+import { useCallback, useEffect, useState } from 'react';
+
+import SuiApp, { SuiAppEmpty } from './SuiApp';
+import Button from '_app/shared/button';
+import ExplorerLink from '_components/explorer-link';
+import { ExplorerLinkType } from '_components/explorer-link/ExplorerLinkType';
+import Icon, { SuiIcons } from '_components/icon';
+import LoadingIndicator from '_components/loading/LoadingIndicator';
+import { useAppSelector, useAppDispatch } from '_hooks';
+import { getCuratedApps } from '_redux/slices/dapps';
+import { mintDemoNFT } from '_redux/slices/sui-objects';
+
+import type { SerializedError } from '@reduxjs/toolkit';
+
+import st from './Playground.module.scss';
+
+function AppsPlayGround() {
+    const [mintInProgress, setMintInProgress] = useState(false);
+    const [mintStatus, setMintStatus] = useState<boolean | null>(null);
+    const [mintError, setMintError] = useState<string | null>(null);
+    const dispatch = useAppDispatch();
+
+    useEffect(() => {
+        dispatch(getCuratedApps()).unwrap();
+        let timeout: number;
+        if (mintStatus !== null) {
+            timeout = window.setTimeout(() => setMintStatus(null), 3000);
+        }
+        return () => {
+            if (timeout) {
+                clearTimeout(timeout);
+            }
+        };
+    }, [dispatch, mintStatus]);
+
+    const curatedDapps = useAppSelector(
+        ({ curatedApps }) => curatedApps.curatedApps
+    );
+
+    const handleMint = useCallback(async () => {
```

Comment:
It'd probably be good to introduce [TanStack Query](https://tanstack.com/query/v4) to the wallet as well to clean up this mutation code. We can also just open an issue to do this later.